dev-client/src/components/home/BottomSheet.tsx

```diff
@@ -190,13 +201,21 @@ const SiteFilterModal = ({useDistance}: ModalProps) => {
         />
       }>
       <SelectFilter
-        label={t('site.search.sort.label')}
+        label={t('site.search.filter_projects') + ':'}
+        options={projectOptions}
+        name="project"
+        nullableOption={t('general.filter.no_project')}
+      />
+
+      <SelectFilter
+        label={t('site.search.sort.label') + ':'}
         options={sortOptions}
         name="sort"
         nullableOption={t('general.filter.no_sort')}
       />
+
       <SelectFilter
-        label={t('site.search.filter_role')}
+        label={t('site.search.filter_role') + ':'}
```

Comment:
issue: the colon should be in the i18n string, because it needs localizing, too. i.e. in French we may need a space before the colon.

I see that it was removed — what was the reasoning for that?

Reply:
Oh, I didn't know that. I saw it was inconsistent, with some strings having a colon and others not, so I added it here.